pkg/http/core/kubernetes/deploy.go

```diff
@@ -203,6 +213,12 @@ func Deploy(c *gin.Context, dm DeployManifestRequest) {
 			Cluster:      cluster(meta.Kind, name),
 		}
 
+		pipelineArtifacts[meta.Name] = clouddriver.TaskCreatedArtifact{
+			Name:      meta.Name,
+			Reference: artifactName,
```

Comment:
Isn't `meta.Name` going to contain the versioned string at the end? In your test the name references the artifact name without the version and the reference has the version.

Artifact name is defined as follows:
```go
		name := u.GetName()
		artifactName := name
```